README.md

```diff
@@ -88,6 +88,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JavaServer Pages](https://jsp.java.net/) - Common templating for websites with custom tag libraries.
 * [JUnit](http://junit.org/) - Common testing framework.
 * [Launch4j](http://launch4j.sourceforge.net/) - Wraps JARs in lightweight and native Windows executables.
+* [Quartz](https://github.com/quartz-scheduler/quartz) - a richly featured, open source job scheduling library that can be integrated within virtually any Java application - from the smallest stand-alone application to the largest e-commerce system.
```

Comment:
Begin the description with the first letter in uppercase. The description sounds more than an advertisement, please write something more informative.